skimage/morphology/selem.py

```diff
@@ -29,32 +32,46 @@ def square(width, dtype=np.uint8):
     return np.ones((width, width), dtype=dtype)
 
 
-def rectangle(width, height, dtype=np.uint8):
+@deprecate_kwarg({"height": "ncols", "width": "nrows"},
+                 removed_version="0.20.0")
+def rectangle(nrows=None, ncols=None, dtype=np.uint8):
     """Generates a flat, rectangular-shaped structuring element.
 
     Every pixel in the rectangle generated for a given width and given height
     belongs to the neighborhood.
 
     Parameters
     ----------
-    width : int
-        The width of the rectangle.
-    height : int
-        The height of the rectangle.
+    nrows : int
+        The number of rows of the rectangle.
+    ncols : int
+        The number of columns of the rectangle.
 
     Other Parameters
     ----------------
     dtype : data-type
         The data type of the structuring element.
+    height : int, depreciated
+        The number of columns of the rectangle.
+    width : int, depreciated
+        The number of rows of the rectangle.
```

Comment:
```suggestion

```

Reply:
I'm not sure if this should be removed in this PR or at a later date. We are changing 10-year-old code, so some folks may wonder what happened to `width` and `height` and it seems polite to at least mention that they have been deprecated inside the docstring. Hence, I would remove this once skimage moves to a version that removes `width` and `height`.

Perhaps we could add a note instead of these parameters? I do admit that it depends on how you guys handle deprecations.

Reply:
I prefer a note since we are deprecating these parameters and don't want new users to use them...